crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -293,7 +348,17 @@ def upgrade_controller_nodes
         # and the existing cluster has been cleaned up by deleting most of resources
         return false unless node_api.post_upgrade
 
-        # FIXME: if upgrade went well, continue with next node(s)
+        # Delete router namespaces on non-upgraded node
+        master_node_api.delete_router_namespace
```

Comment:
In the recipe, that would be easy, I agree. But how do we create it easily once we need it? We did not plan second chef-client run on the first node